src/components/Doc/ComponentDoc.js

```diff
@@ -49,6 +50,22 @@ export const ComponentDoc = ({
         <title>{pageTitle || name}</title>
         <meta name="description" content={description} />
       </Helmet>
+      {stable && (
+        <Box pad={{ bottom: 'medium' }}>
```

Comment:
Potentially use Notification component here as an inline notification?

Reply:
Changed this to use the notification component in latest commit